### ISSUE TITLE
GA4 for Orcahome

### ISSUE THREAD

**Issue by UXBrendan** [orcasound/orcahome#130]
Is Orcahome currently instrumented for GA4? Or should we rely on analytics from WordPress for until we migrate to the redesign?

**Comment by paulcretu** [comment]
No, it's not set up as far as I know. Definitely something to do before release. It's not hard to add, and actually it could be a great first issue (though it does require GA account access). Wordpress analytics are separate so they're only useful for the existing orcasound.net homepage, they won't provide any data on the project in this repo.

**Comment by UXBrendan** [comment]
It looks like Google automatically ported over analytics of Orcasite to GA4- see image. I've been able to pull current data. Is this accurate do you think?

![Screenshot 2023-12-12 at 12 21 43 PM](https://github.com/orcasound/orcahome/assets/65036138/9fdc8e3b-8cc4-497a-9aa4-49292dc51d0f)
